scripts/setup/rewrite_yaml_files.sh

```diff
@@ -0,0 +1,319 @@
+#!/bin/bash
+
+KNATIVE_VERSION="1.9.0"
+master_node_name=$(hostname)
+MASTER_NODE_IP=$(ip route | awk '{print $(NF)}' | awk '/^10\..*/')
+IFACE=$(netstat -ie | grep -B1 $MASTER_NODE_IP | head -n1 | awk '{print $1}' | cut -d ':' -f 1)
+
+# we set these limits high enough but to fit in the budget of a typical master node server
+cpu_limit_net_istio=2
+memory_limit_net_istio="10Gi"
+cpu_limit_serving_core=3
+memory_limit_serving_core="10Gi"
+
+pushd $HOME/vhive/configs/knative_yamls >/dev/null
+
+rm serving-core-firecracker.yaml
+rm serving-default-domain.yaml
+rm serving-hpa.yaml
+rm serving-post-install-jobs.yaml
+rm serving-storage-version-migration.yaml
+rm ../metallb/metallb-ipaddresspool.yaml
+rm ../setup/kube.json
+
+wget -q https://github.com/knative-extensions/net-istio/releases/download/knative-v${KNATIVE_VERSION}/net-istio.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-core.yaml && mv serving-core.yaml serving-core-firecracker.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-core.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-crds.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-default-domain.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-hpa.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-post-install-jobs.yaml
+wget -q https://github.com/knative/serving/releases/download/knative-v${KNATIVE_VERSION}/serving-storage-version-migration.yaml
+
+# net-istio.yaml
+
+cat net-istio.yaml |
+    yq '
+    (
+        select
+        (
+               .spec.selector.matchLabels.app == "net-istio-controller"
+            or .spec.selector.matchLabels.app == "net-istio-webhook"
+        )
+        | .spec.template.spec.containers[0].resources.limits.cpu
+    ) = '"${cpu_limit_net_istio}"' ' |
+    yq '
+    (
+        select
+        (
+               .spec.selector.matchLabels.app == "net-istio-controller"
+            or .spec.selector.matchLabels.app == "net-istio-webhook"
+        )
+        | .spec.template.spec.containers[0].resources.limits.memory
+    ) = "'"${memory_limit_net_istio}"'" ' |
+    # Delete spec.template to prevent parameters from being overwritten
```

Comment:
Didn't quite get this part. Where did you get this explanation? I've checked with the file from hy branch, these lines are there. Why are you deleting it?